primitives/npos-elections/src/tests.rs

```diff
@@ -72,6 +72,50 @@ fn float_phragmen_poc_works() {
 	);
 }
 
+#[test]
+fn phragmen_core_setup_inputs() {
```

Comment:
This test case is not testing the change of this PR. The logic of this PR is to see what happens when there is a voter with no edges. You are not checking this here.

Reply:
(10, 10, vec![1, 2]) 
@kianenigma you mean check with blank vector. something like this (10,10, vec![]). Right?